src/javascript/JContent/ContentRoute/MainActionBar/MainActionBar.jsx

```diff
@@ -29,7 +29,7 @@ export const MainActionBar = () => {
 
     return (
         <div className={styles.root}>
-            <DisplayAction actionKey="search" context={{path, disabled}} render={ButtonRenderer} size="big" variant="ghost"/>
+            <DisplayAction actionKey="search" context={{path, disabled}} render={ButtonRenderer} size="big" variant="ghost" data-sel-role="open-search-dialog"/>
```

Comment:
Is this ok with the changes from https://github.com/Jahia/javascript-components/commit/3e4be47705ef8573ad2a0008e8219e8660eaf8c2 ?